mat64/symmetric.go

```diff
@@ -237,7 +236,30 @@ func (s *SymDense) SymRankK(a Symmetric, alpha float64, x Matrix) {
 //  s = x * x'
 func (s *SymDense) SymOuterK(x Matrix) {
 	r, _ := x.Dims()
-	s.reuseAs(r)
+
```

Comment:
Yeah, I tried that and list test said that s was unmodified in some cases (it must have been too late).